public/app/plugins/datasource/cloudwatch/datasource.test.ts

```diff
@@ -264,7 +252,7 @@ describe('datasource', () => {
       expect(emits[0].data[0].fields.find((f: Field) => f.name === '@message').config.links).toMatchObject([
         {
           title: 'View in CloudWatch console',
-          url: "https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#logs-insights:queryDetail=~(end~'2020-12-31T19*3a00*3a00.000Z~start~'2020-12-31T19*3a00*3a00.000Z~timeType~'ABSOLUTE~tz~'UTC~editorString~'some*20query~isLiveTail~false~source~(~'test))",
```

Comment:
Changing `addDataLinksToLogsResponse` to use the request range instead of the `timeSrv` range changes this test, but I'd argue that we should have been using the request range to begin with.